cell.js

```diff
@@ -374,13 +382,23 @@
 
   // For testing
   if (typeof exports !== 'undefined') {
-    var x = { Phenotype: Phenotype, Genotype: Genotype, Nucleus: Nucleus, Gene: Gene, Membrane: Membrane, God: God };
+    var x = {
+      Phenotype: Phenotype,
+      Genotype: Genotype,
+      Nucleus: Nucleus,
+      Gene: Gene,
+      Membrane: Membrane,
+      God: God,
+      create: God.create.bind(God),
```

Comment:
allows `cell.create(some dom window)` instead of `cell.God.create(some dom window)`